src/assets/styles.scss

```diff
@@ -182,6 +182,37 @@ hr {
     margin: 0.6em 0em;
 }
 
+/* Index.vue page B-card styling */
+
+.index2-row {
```

Comment:
Please move than in the `index-container` class.

```
.index-container {
    <some css components>
    .index2-row {
        <some css components>
    }
}
```

Same for the other classes below

Reply:
okay...and i'll rename it to `index-row2`